jscomp/ml/translcore.ml

```diff
@@ -350,6 +365,19 @@ let primitives_table =
       ("%lefloat", Pfloatcomp Cle);
       ("%gtfloat", Pfloatcomp Cgt);
       ("%gefloat", Pfloatcomp Cge);
+      ("%negbigint", Pnegbigint);
+      ("%addbigint", Paddbigint);
+      ("%subbigint", Psubbigint);
+      ("%mulbigint", Pmulbigint);
+      ("%divbigint", Pdivbigint Safe);
+      ("%powbigint", Ppowbigint);
+      ("%modbigint", Pmodbigint Safe);
+      ("%eqbigint", Pbigintcomp Ceq);
+      ("%noteqbigint", Pbigintcomp Cneq);
+      ("%ltbigint", Pbigintcomp Clt);
+      ("%lebigint", Pbigintcomp Cle);
+      ("%gtbigint", Pbigintcomp Cgt);
+      ("%gebigint", Pbigintcomp Cge);
```

Comment:
It seems `%lslbigint`(`<<`), `%lsrbigint`(`>>`), `%andbigint`(`&`), `%orbigint`(`|`), `%xorbigint`(`^`) are missing? (not sure bitwise not op) Logical operators are the main interface of BigInt.

Note that ASL(Arithmetic Shift Right, aka Unsigned right shift in JS) doesn't accept BigInt.

Reply:
Initially, I wasn't going to include the bitwise operation implementation in this PR because I was concerned that it would make the PR too large, but now that I think about it, it doesn't make much difference, so I'll add it.

Reply:
Added https://github.com/rescript-lang/rescript-compiler/pull/6670/commits/9060aa0e4fe40aafd7ca0605c6e6603890093d37